examples/03-advanced/03-exchange_data_between_servers.py

```diff
@@ -8,12 +8,20 @@
 with a part on both servers. This example shows how you can read data
 from a given machine and transform this data on another machine
 without any more difficulties than working on a local computer.
+
+.. note::
+    This example requires the Premium ServerContext.
+    For more information, see :ref:`_ref_getting_started_contexts`.
```

Comment:
do we need premium here?

Reply:
@cbellot000 geo.rotate_in_cylindrical_cs_fc doesn't exist when Entry